checkstyle.xml

```diff
@@ -0,0 +1,250 @@
+<?xml version="1.0"?>
```

Comment:
next time you can just use the link for the checkstyle instead of creating separate file